cibyl/sources/jenkins.py

```diff
@@ -307,6 +312,75 @@ def get_last_build(self, **kwargs):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_objects)
 
+    def get_tests(self, **kwargs):
```

Comment:
The get_tests is missing a speed index, just add `@speed_index({'base': 2})` to decorate the get_tests methods and should be good to go, we can ajust the actual value as more sources implement it and we can compare.

Reply:
Done, thanks